src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MainConsumerImpl.java

```diff
@@ -132,15 +132,19 @@ public void readAndProcessRetryTopic(@Payload List<ChipsRestInterfacesSend> mess
                 failedMessageIds.add(messageID);
 
                 acknowledgment.nack(i, batchFailureRetrySleepMs);
-                logger.infoContext(messageID, String.format("Committed messages in batch up to offset %s", offset), logMap);
+                logger.infoContext(messageID, String.format("Acknowledging (committing) messages in batch up to but not including offset %s", offset), logMap);
```

Comment:
I would used past tense "Acknowledged (committed) ..." here to be consistent with other logs.
And the action has happened by this point

Reply:
updated